### ISSUE TITLE
Storybook accessibility: code should have sufficient color contrast

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#5903]
### Expected Behavior
Our storybook should meet accessibility requirements.

### Actual Behavior
Grommet's storybook isn't passing accessibility checks. The code examples don't have enough contrast. We should see if it is possible to change the code example colors.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io

### Steps to Reproduce

1. Go to https://storybook.grommet.io
2. Run axe DevTools
